docs/guides/server/caching.adoc

```diff
@@ -91,66 +91,58 @@ CPU, memory, and network utilization.
 
 User and client sessions are automatically destroyed whenever the user performs a logout, the client performs a token revocation, or due to reaching their expiration time.
 
-The following caches are used to store both user and client sessions:
+The session data are stored in the database by default and loaded on-demand to the following caches:
 
 * sessions
 * clientSessions
 
-By relying on a distributable cache, user and client sessions are available to any node in the cluster so that users can be redirected
-to any node without losing their state. However, production-ready deployments should always consider session affinity and favor redirecting users
+By relying on a distributable cache, cached user and client sessions are available to any node in the cluster so that users can be redirected
+to any node without the need to load session data from the database. However, production-ready deployments should always consider session affinity and favor redirecting users
 to the node where their sessions were initially created. By doing that, you are going to avoid unnecessary state transfer between nodes and improve
 CPU, memory, and network utilization.
 
-ifeval::[{project_community}==true]
+These in-memory caches for user sessions and client sessions are limited to, by default, 10000 entries per node which reduces the overall memory usage of {project_name} for larger installations.
+The internal caches will run with only a single owner for each cache entry.
+Consider trade-off between memory consumption and the database utilization and set different sizes for the caches, edit {project_name}'s cache config file (`conf/cache-ispn.xml`) to set a `+<memory max-count="..."/>+` for those caches.
+
+.Volatile user sessions
+
+By default, user sessions are stored in the database and loaded on-demand to the cache.
+It is possible to configure {project_name} to store user sessions in the cache only and minimize the database utilization.
 
-.Persistent user sessions
-The feature `persistent-user-sessions` stores online user and client sessions also in the database.
-This will allow a user to stay logged in even if all instances of {project_name} are restarted or upgraded.
+Since all the sessions in this setup are stored in-memory, there are two side effects related to this:
+* Losing sessions on all Keycloak nodes restart
+* Increased memory consumption
 
-The feature is disabled by default. To use it, enable the feature:
+Follow these steps to enable this setup:
 
+1. Configure caches to not limit number of entries, as the cache will be the only source of truth for user and client sessions. To do so, edit {project_name}'s cache config file (`conf/cache-ispn.xml`) to remove the `+<memory max-count="..."/>+` for the `sessions` and `clientSessions` caches.
```

Comment:
It needs another step to set the `owner=2` (or more). It is `1` in KC default configuration.